src/badge.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('material-design-lite/material.min.css');
+
+const React = require('react');
+const mdl = require('material-design-lite/material.min');
+const classnames = require('classnames');
+
+const baseClasses = {
+  'mdl-badge': true
+};
+
+class Badge extends React.Component {
+
+  componentDidMount(){
+    const node = React.findDOMNode(this);
+    mdl.upgradeElement(node, 'MaterialBadge');
+  }
+
+  componentWillUnmount(){
+    const node = React.findDOMNode(this);
+    mdl.downgradeElements(node);
+  }
+
+  render(){
+    const {
+      href = '#',
```

Comment:
maybe it shouldn't default at all, I feel like `#` is an anti-pattern in react since everything is wired up on render